datalad/metadata/aggregate.py

```diff
@@ -150,11 +151,14 @@ def _extract_metadata(agginto_ds, aggfrom_ds, db, to_save):
     # 1. the latest commit that changed any file for which we could have native metadata
     refcommit = _get_latest_refcommit(aggfrom_ds, subds_relpaths)
     objid = refcommit if refcommit else ''
-    # 2, our own dataset-global metadata
-    dsmetafile = opj(aggfrom_ds.path, '.datalad', 'metadata', 'dataset.json')
-    if exists(dsmetafile):
-        objid += md5(open(dsmetafile, 'r').read().encode()).hexdigest()
+    # 2, our own dataset-global metadata and the dataset config
+    for tfile in (
+            opj(aggfrom_ds.path, '.datalad', 'metadata', 'dataset.json'),
+            opj(aggfrom_ds.path, '.datalad', 'config')):
+        if exists(tfile):
+            objid += md5(open(tfile, 'r').read().encode()).hexdigest()
```

Comment:
is there a need to have both digests concatenated within the objid (longish filename, more to commit to git) or it could be a either hexdigest on a collection of digests (easier), or just hexdigest of concatenated content?

we already have  datalad.utils:md5sum
```
datalad/utils.py-
datalad/utils.py-def md5sum(filename):
datalad/utils.py-    with open(filename, 'rb') as f:
datalad/utils.py:        return hashlib.md5(f.read()).hexdigest()
```

Reply:
If you skip a few more lines down, I'll see that it finally get's condensed into a single short MD5sum.